src/commands/dev-env-sync-sql.js

```diff
@@ -166,6 +166,8 @@ export class DevEnvSyncSQLCommand {
 		try {
 			await this.generateExport();
 		} catch ( err ) {
+			// this.generateExport probably catches all exceptions and runs exit.withError() already but just in case, let's log it here
```

Comment:
Is there a chance that we'd be duplicating the event? That would create bad data?

Reply:
There's a small chance because we're relying on developer discipline to `try catch` at each place and making sure it exits rather than, say, catching everything in one place and then exit.

From reading the code, the developer discipline that we have is good right now so this is safe.

More than that, we'd actually want it to trigger twice, because that'd mean that something went really wrong and someone made it trigger twice i.e. by throwing twice, or if `exit.withError` failed badly, or if `trackEvent` somehow threw an exception too!

Reply:
I meant that we should not trigger the `export_sql_backup` event itself twice in the same flow . And I don't think we're doing that so we should be good.

When we trigger it twice (or duplicate it essentially) we create bad data for analysers who look at the number of errors we received for a tool and gauge whether the tool is performing well or not.

Reply:
Oh, all right. Correct, we're not triggering that particular event twice regardless.